docs/pages/application-access/getting-started.mdx

```diff
@@ -21,9 +21,21 @@ Let's connect to Grafana using Teleport Application Access in three steps:
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-- (!docs/pages/includes/tctl.mdx!)
+(!docs/pages/includes/tctl.mdx!)
```

Comment:
This line shouldn't have been changed. We've updated the tctl partial to fit as a UL item.

Reply:
```suggestion
- (!docs/pages/includes/tctl.mdx!)
```